src/main/java/javaslang/collection/List.java

```diff
@@ -1348,6 +1354,32 @@ public T head() {
         }
 
         @Override
+        public Tuple2<List<T>, List<T>> splitAt(Predicate<? super T> predicate) {
+            Tuple2<List<T>, List<T>> t = splitByPredicateFirstRound(this, predicate);
```

Comment:
If possible, generally please make all values final. It is one of the main rules to achieve functional programs.